react/typings/vtex.styleguide.d.ts

```diff
@@ -1,9 +1 @@
-declare module 'vtex.styleguide' {
-  import { ComponentType } from 'react'
-
-  export const Input: ComponentType<InputProps>
-
-  interface InputProps {
-    [key: string]: any
-  }
-}
+declare module 'vtex.styleguide'
```

Comment:
can you use [module augmentation](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) for this? so we can benefit of all the types that exist in the styleguide package